app/config.py

```diff
@@ -15,8 +15,7 @@ class Config(object):
     HEADER_COLOUR = '#81878b'  # mix(govuk-colour("dark-grey"), govuk-colour("mid-grey"))
     LOGO_CDN_DOMAIN = 'static-logos.notifications.service.gov.uk'  # TODO use our own CDN
     ASSETS_DEBUG = False
-    TIMEZONE = getenv('TIMEZONE', 'America/New_York')
-    PY_TIMEZONE = pytz.timezone(TIMEZONE)
+    PY_TIMEZONE = pytz.utc
```

Comment:
Any reason to keep `PY_TIMEZONE`? Seems like that one might be able to go, too.

Reply:
I removed `TIMEZONE` but not `PY_TIMEZONE` since I saw `PY_TIMEZONE` referenced in quite a few different places.  We could just use `pytz.utc` at this point in all of places it's found at this point, but I'd save it for a cleanup PR to make sure we don't miss anything.